docs/core/getting_started/installation.md

```diff
@@ -53,17 +54,25 @@ Examples of extra packages include:
 
 ## Running the local server and UI
 
-In addition to the [Prefect Cloud](https://www.prefect.io/cloud) platform, Prefect includes an open-source server and UI for orchestrating and managing flows. The local server stores flow metadata in a Postgres database and exposes a GraphQL API.
+Prefect includes an open-source server and UI for orchestrating and managing flows. The local server stores flow metadata in a Postgres database and exposes a GraphQL API. The local server requires [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/) to be installed. If you have [Docker Desktop](https://www.docker.com/products/docker-desktop) on your machine, you've got both of these.
```

Comment:
@jcrist  There ya go, `running` --> `installed`. I also edited the next sentence so that the word `installing` does get tediously repeated.